README.md

```diff
@@ -43,6 +42,8 @@ To run the container in demo mode, override the `GROCY_MODE` environment variabl
 GROCY_MODE=demo docker-compose up
 ```
 
+The default login credentials are username `admin` and password `admin`; please change these before providing end-user access to your deployment.
```

Comment:
Let's keep this paragraph within the `Configuration` heading?  The default credentials apply whether or not the container runs in `demo` mode or not.

Reply:
I understand what you mean. I thought the install narrative made more sense to have the login credentials be the last step, but that it could be confused as just for the demo mode is definitely there. Ill leave that to your discretion.